pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "foursight"
-version = "2.1.1"
+version = "2.2.0.1b58"
```

Comment:
This should be based on version  it came from + epsilon, not on a guess of where it's going + epsilon, since that is an overshoot and means the release you want would be lower than this.  That can't work. See explanation at [semantic versioning](https://hms-dbmi.atlassian.net/wiki/spaces/FOURDNDCIC/pages/1107525651/Understanding+Semantic+Versions) in the Managing Beta Versions part of the page.

It also feels like the b58 designation relates to stuff that's been merged to master and should start again from b0.

Reply:
Oh, thanks, I had that backward, will change to 2.1.1.1b0 (right?). Means I polluted the 2.2.0 beta version space, sorry about that.